app/controllers/root_controller.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+class RootController < ApplicationController
```

Comment:
is `Root` the right noun to describe whatever this resource is?

Reply:
Suggestions welcome :pray: 

from  route.rb
```ruby
root to: "root#start"
```

from usage
```
root_path
```

actually location is also `/`

I did consider `HomeController` but felt stumped by 
```
views/home/home.html.erb
views/home/start.html.erb
```

Reply:
Discussed and we settled on `LandingPage` vs `Home` vs `Dashboard`